src/core/thread/mlr_manager.hpp

```diff
@@ -113,8 +130,34 @@ class MlrManager : public InstanceLocator, public Notifier::Receiver
                                                      const Ip6::MessageInfo *aMessageInfo,
                                                      otError                 aResult);
 
-    uint16_t CountNetifMulticastAddressesToRegister(void) const;
-    void     SetNetifMulticastAddressMlrState(MlrState aFromState, MlrState aToState);
+#if OPENTHREAD_CONFIG_MLR_ENABLE
+    void UpdateLocalSubscriptions(void);
+    void SetNetifMulticastAddressMlrState(MlrState aFromState, MlrState aToState);
+    bool IsAddressMlrRegisteredByNetif(const Ip6::Address &aAddress);
```

Comment:
nit: declare as `const`?
```suggestion
    bool IsAddressMlrRegisteredByNetif(const Ip6::Address &aAddress) const;
```